backend/src/test/java/gov/cdc/usds/simplereport/api/BaseApiTest.java

```diff
@@ -3,133 +3,107 @@
   @Autowired protected DemoOktaRepository _oktaRepo;
   @Autowired protected GraphQLTestTemplate _template; // screw delegation
-  @MockBean protected AuthorizationService _authService;
-  @MockBean protected IdentitySupplier _supplier;
-
-  private static final List<OrganizationRoleClaims> USER_ORG_ROLES =
-      Collections.singletonList(
-          new OrganizationRoleClaims("DIS_ORG", Set.of(OrganizationRole.USER)));
-  private static final List<OrganizationRoleClaims> OUTSIDE_USER_ORG_ROLES =
-      Collections.singletonList(
-          new OrganizationRoleClaims("DAT_ORG", Set.of(OrganizationRole.USER)));
-  private static final List<OrganizationRoleClaims> ADMIN_ORG_ROLES =
-      Collections.singletonList(
-          new OrganizationRoleClaims(
-              "DIS_ORG", Set.of(OrganizationRole.USER, OrganizationRole.ADMIN)));
-  private static final List<OrganizationRoleClaims> OUTSIDE_ADMIN_ORG_ROLES =
-      Collections.singletonList(
-          new OrganizationRoleClaims(
-              "DAT_ORG", Set.of(OrganizationRole.USER, OrganizationRole.ADMIN)));
-  private static final List<OrganizationRoleClaims> ENTRY_ONLY_ORG_ROLES =
-      Collections.singletonList(
-          new OrganizationRoleClaims(
-              "DIS_ORG", Set.of(OrganizationRole.USER, OrganizationRole.ENTRY_ONLY)));
+  @Autowired private DemoUserConfiguration _users;
+
+  private String _userName = null;
 
   protected void truncateDb() {
     _truncator.truncateAll();
   }
 
   protected void useOrgUser() {
-    LoggerFactory.getLogger(BaseApiTest.class).info("Configuring auth service mock for org user");
-    when(_supplier.get()).thenReturn(TestUserIdentities.STANDARD_USER_ATTRIBUTES);
-    when(_authService.findAllOrganizationRoles()).thenReturn(USER_ORG_ROLES);
-    _initService.initCurrentUser();
+    LOG.info("Configuring auth service mock for org user");
+    _userName = TestUserIdentities.STANDARD_USER;
   }
 
   protected void useOutsideOrgUser() {
-    LoggerFactory.getLogger(BaseApiTest.class)
-        .info("Configuring auth service mock for outside org user");
-    when(_supplier.get()).thenReturn(TestUserIdentities.STANDARD_USER_ATTRIBUTES);
-    when(_authService.findAllOrganizationRoles()).thenReturn(OUTSIDE_USER_ORG_ROLES);
-    _initService.initCurrentUser();
+    LOG.info("Configuring auth service mock for outside org user");
+    _userName = "intruder@pirate.com";
   }
 
   protected void useOrgAdmin() {
-    LoggerFactory.getLogger(BaseApiTest.class).info("Configuring auth service mock for org admin");
-    when(_supplier.get()).thenReturn(TestUserIdentities.STANDARD_USER_ATTRIBUTES);
-    when(_authService.findAllOrganizationRoles()).thenReturn(ADMIN_ORG_ROLES);
-    _initService.initCurrentUser();
+    _userName = "admin@example.com";
+    LOG.info(
+        "Configuring auth service mock for org admin: {}",
+        _users.getByUsername(_userName).getAuthorization().getGrantedPermissions());
```

Comment:
actually i'm confused, you're logging the permissions here? why?

Reply:
Debugging the test framework. Deleted in the still-unpushed clean-up commit,  IIRC.